Assets/MRTK/Core/Providers/InputSimulation/SimulatedHandDataProvider.cs

```diff
@@ -27,10 +27,9 @@ internal class SimulatedHandState
         // Show a tracked hand device
         public bool IsTracked = false;
         // Activate the pinch gesture
-        public bool IsPinching
-        {
-            get { return gesture == ArticulatedHandPose.GestureId.Pinch; }
-        }
+        // Pinch is a special gesture that triggers the Select and TriggerPress input actions
+        // The pinch action doesn't occur until the gesture is completed.
+        public bool IsPinching => gesture == ArticulatedHandPose.GestureId.Pinch && gestureBlending == 1.0f;
```

Comment:
It looks like after making this change, hands-free input stopped working. Seems like our regression tests did not catch this.